docker-compose.yml

```diff
@@ -52,3 +52,21 @@ services:
     depends_on:
       - elasticsearch
 
+  composer:
+    image: composer:1.10
```

Comment:
IMO, la v2 es the way to go

Reply:
La verdad es que si. Puedo hacer algunas pruebas y ver si no hay problemas con los paquetes!

Reply:
Subido a versión 2 y hechas pruebas en local, todo parece funcionar correctamente, gracias por la sugerencia :)